core/admin/assets/ee-admin-page.css

```diff
@@ -74,6 +74,7 @@
 
 .poststuff .ee-nav-tabs-small .nav-tab,
 #poststuff .ee-nav-tabs-small .nav-tab {
+    box-sizing: content-box;
```

Comment:
is intendation changed?

![image](https://user-images.githubusercontent.com/1477453/105755091-81d3d580-5f53-11eb-9648-f7c325f67a1f.png)

Reply:
nope... thnx, will fix